tgui/packages/tgui/interfaces/Orbit/index.tsx

```diff
@@ -0,0 +1,236 @@
+import { filter, sortBy } from 'common/collections';
+import { flow } from 'common/fp';
+import { capitalizeFirst } from 'common/string';
+import { useBackend, useLocalState } from 'tgui/backend';
+import { Button, Collapsible, Icon, Input, LabeledList, Section, Stack } from 'tgui/components';
+import { Window } from 'tgui/layouts';
+import { getDisplayColor, getDisplayName, isJobOrNameMatch } from './helpers';
+import type { Observable, OrbitData } from './types';
+
+export const Orbit = (props, context) => {
+  return (
+    <Window title="Orbit" width={400} height={550}>
```

Comment:
preserve old orbit menu's width and height (I changed it because we get so many players it was hard to use when small)